lib/components/common/NoResultField.tsx

```diff
@@ -1,38 +1,39 @@
-import { Button, Typography, Box } from '@mui/material';
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them